nemo-wasm/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Is this really still needed? Since this is now part of the workspace, artifacts should go in the top-level `target` directory, which is already ignored.